Dockerfile.prow

```diff
@@ -6,7 +6,7 @@ WORKDIR /workspace
 COPY . .
 
 RUN go mod vendor
-RUN go install github.com/prometheus/promu@v0.17.0 && make common-build
+RUN go install github.com/prometheus/promu@v0.17.0 && promu build -v --cgo --prefix ./
```

Comment:
Same here, why not just add the flag to the build target instead of separately using promu

Reply:
CI / Build Alertmanager for common architectures (1) (pull_request)  these builds fail when I add the CGO flags